examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -1207,14 +1206,15 @@ def compute_time_ids(original_size, crops_coords_top_left):
             text_encoder_2_lora_layers=text_encoder_2_lora_layers,
         )
 
+        del unet
+        del text_encoder_one
+        del text_encoder_two
+        del text_encoder_lora_layers
+        del text_encoder_2_lora_layers
+        torch.cuda.empty_cache()
+
         # Final inference
         # Load previous pipeline
-        vae = AutoencoderKL.from_pretrained(
```

Comment:
is this already defined?

Reply:
Yes, we never del the one defined [here](https://github.com/huggingface/diffusers/blob/fed88195e363689d92c5626e48ced31d007a67be/examples/text_to_image/train_text_to_image_lora_sdxl.py#L565) so is still there even at the end. And... it works without it :)